.github/workflows/main.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       env:
         COVERAGE: false
         # The value doesn't matter as long as it's > 2.087
-        FRONTEND: 2.093.0
+        FRONTEND: 2.095.0
```

Comment:
Those might get out of sync though

Reply:
They *did* get out of sync - it seems to be an environment variable though, so I didn't see a simple way of doing it automatically at 5AM